pipeline/outputs/s3.md

```diff
@@ -282,6 +289,23 @@ Example:
 
 Then, the records will be stored into the MinIO server.
 
+## Usage for Parquet Compression
+
+For parquet compression, it needs to install [columnify](https://github.com/reproio/columnify) at runtime.
```

Comment:
What does this mean? Is it in the image or part of the package? Or do we have to do it separately?

Reply:
It means that columnify command should be installed in the system or container at runtime. This is not building dependency but for runtime dependency.

Reply:
Added in https://github.com/fluent/fluent-bit-docs/pull/1380/commits/0b6238b03597107d8727c82557635a77a57e03db.